views/swap/input/dollar-value.tsx

```diff
@@ -28,12 +28,8 @@ const AmountInDollar: FC<InputProps> = ({ label }) => {
       variant="body"
       color={value ? 'onSurface' : 'outline'}
     >
-      {usdPrice
-        ? formatDollars(
-            BigNumber(value || 0)
-              .times(BigNumber(usdPrice))
-              .toNumber()
-          )
+      {usdPrice && value
```

Comment:
For some reason, it's not working properly

Reply:
what you mean with display, can you send a screenshot?

Reply:
<img width="581" alt="Screenshot 2024-02-13 at 19 27 14" src="https://github.com/interest-protocol/sui-coins/assets/37873239/129d6ffd-5a44-4864-9712-b262b54f44fb">


USD price is not displayed